Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -176,6 +176,7 @@ const Info<std::string> MAIN_RESOURCEPACK_PATH{{System::Main, "General", "Resour
 const Info<std::string> MAIN_FS_PATH{{System::Main, "General", "NANDRootPath"}, ""};
 const Info<std::string> MAIN_SD_PATH{{System::Main, "General", "WiiSDCardPath"}, ""};
 const Info<std::string> MAIN_WFS_PATH{{System::Main, "General", "WFSPath"}, ""};
+const Info<std::string> MAIN_GECKO_URL{{System::Main, "General", "GeckoURL"}, ""};
```

Comment:
This should probably have something like "download" in the name, since otherwise it gives the impression of being used by the actual Gecko implementation.

Reply:
Refering to "GeckoURL" or MAIN_GECKO_URL?

Reply:
Both.

Reply:
Ok